### ISSUE TITLE
Unable to see type annotations 0.44 OSX

### ISSUE THREAD

**Issue by srsholmes** [reasonml-editor/reasonml-idea-plugin#43]
plugin version: 0.44

Hello, firstly i'd like to say thanks for all your efforts in making this plugin, it's really coming along well, and we appreciate it a lot, so thanks for that 👍 .

I am having trouble getting the type annotations to display in Webstorm. 

I have tried to set my custom VM options to many different variations, with no luck (however the Bucklescript plugin will complain if the path is not found, so i guess it is finding it correctly).  I get no error when i omit the paths from the options (and it defaults to the node modules). 

Currently i have it set to the following:

`-DreasonBsb=./node_modules/bs-platform/lib/bsb.exe`
`-DreasonRefmt=./node_modules/bs-platform/lib/refmt.exe`

 but i have also tried my global installation of Bucklescript in nvm, with no luck. 

I also thought it might be my theme settings, but the color scheme does show them, so i would assume they should show. 

<img width="1552" alt="screen shot 2018-04-05 at 09 09 20" src="https://user-images.githubusercontent.com/3579905/38354531-d63f1182-38b1-11e8-9e53-e5641d0d90dc.png">

As you can see the Bucklescript plugin is working 

<img width="1552" alt="screen shot 2018-04-05 at 09 08 30" src="https://user-images.githubusercontent.com/3579905/38354556-f3114dac-38b1-11e8-818c-af00dbb1ec53.png">

I have the project set up as per the default project described the reasonml docs. 

I have also tried to set the `"namespace": false` and also `"namespace": true` in the bsconfig but neither worked.

Thanks for your help in advance, and keep up the good work!

**Comment by giraud** [comment]
Hey, thanks, always nice to hear happy users :)

You've already done many checks! 
One note, if you install bucklescript locally to your project, you don't need to define the reasonBsb and reasonRefmt properties.

Do check also that you have cmt files in your `lib/bs/src` directory that correspond to your source, for ex `lib/bs/src/Demo.cmt`.
Do check also that the file editor view is associated with the plugin, in `settings | Editor | File Types`.

**Comment by srsholmes** [comment]
Hi @giraud , thanks for getting back to me. 

I do have the .cmt files in my `lib/bs/src`, screen shot below
<img width="1552" alt="screen shot 2018-04-06 at 10 20 11" src="https://user-images.githubusercontent.com/3579905/38413710-5e83a5b2-3984-11e8-9207-19f5754c2b54.png">

and the .re and .rei file types appear to be associated with the file types in the editor. 

<img width="1552" alt="screen shot 2018-04-06 at 10 18 54" src="https://user-images.githubusercontent.com/3579905/38413790-9d6f221a-3984-11e8-87ed-93ce1ddde44d.png">

Thanks for your help. Let me know if there's anything I can do to help debug whats going on, although i dont have much experience with java unfortunately, so may be limited help.

**Comment by srsholmes** [comment]
I installed the plugin, then did a grep for 'ReasonMl' which returned nothing, but then did grep on the logs for "ReasonML", and this is what it came up with 

```
2018-04-06 10:20:06,928 [70953175]   INFO -                       ReasonML - Identifying 'node_modules/bs-platform/lib/refmt.exe' binary
2018-04-06 10:20:06,928 [70953175]   INFO -                       ReasonML - Found './node_modules/bs-platform/lib/refmt.exe' in the property 'reasonRefmt'
2018-04-06 12:08:36,447 [77462694]   INFO - cloudConfig.CloudConfigManager - === installed_plugins.txt: PluginDescriptor[name='ReasonML language plugin', classpath='/Users/cabbio540/Library/Application Support/WebStorm2018.1/reasonml-plugin-idea'] ===
2018-04-06 12:08:36,451 [77462698]   INFO - cloudConfig.CloudConfigManager - === installed_plugins.txt.Remove: PluginDescriptor[name='ReasonML language plugin', classpath='/Users/cabbio540/Library/Application Support/WebStorm2018.1/reasonml-plugin-idea'] ===
2018-04-06 12:09:39,951 [  58678]   INFO - cloudConfig.CloudConfigManager - === InstallPlugin: PluginDescriptor[name='ReasonML language plugin', classpath='/Users/cabbio540/Downloads/reasonml-plugin-idea/lib/reasonml-idea-plugin-0.45-BETA.jar'] ===
2018-04-06 12:09:39,952 [  58679]   INFO - cloudConfig.CloudConfigManager - === installed_plugins.txt.Add: PluginDescriptor[name='ReasonML language plugin', classpath='/Users/cabbio540/Downloads/reasonml-plugin-idea/lib/reasonml-idea-plugin-0.45-BETA.jar'] ===
2018-04-06 12:09:57,587 [   4700]   INFO - llij.ide.plugins.PluginManager - Loaded custom plugins: .ignore (2.5.0), ANSI Highlighter (1.2.4), AceJump (3.5.0), BashSupport (1.6.13.181), Database Navigator (3.0.8071.0), ESLint (1.0.34), GoToTabs (1.3), IdeaVim (0.49.4), Ideolog (181.0.7.0), Ini4Idea (181.3741.23), JS GraphQL (1.7.0), JS Toolbox (1.9), Markdown support (181.2668), Material Theme UI (2.1.4), PostCSS support (181.3494.12), Protobuf Support (0.10.2), Quick Notes (2.9.6), Quokka (1.0.88), ReasonML language plugin (0.45-BETA), Relative Line Numbers (1.0.1), Shifter (1.7.7), String Manipulation (5.7.143.000.0), Styled Components (1.0.5), Tab Shifter (0.25)
2018-04-06 12:10:23,750 [  30863]   INFO -                       ReasonML - Identifying 'node_modules/bs-platform/lib/bsb.exe' binary
2018-04-06 12:10:23,750 [  30863]   INFO -                       ReasonML - Found './node_modules/bs-platform/lib/bsb.exe' in the property 'reasonBsb'
```

Webstorm also reported 3 errors when launching the project with the following: 

```
com.reason.ide.RmlProjectTracker@58d8026a: io/reactivex/subjects/Subject
java.lang.NoClassDefFoundError: io/reactivex/subjects/Subject
	at com.reason.ide.RmlProjectTracker.projectOpened(RmlProjectTracker.java:23)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.e(ProjectManagerImpl.java:770)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.g(ProjectManagerImpl.java:386)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:165)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:447)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:431)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:415)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.m(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at com.intellij.ide.IdeEventQueue.pumpEventsForHierarchy(IdeEventQueue.java:920)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:215)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runProcessWithProgressSynchronously(ApplicationImpl.java:579)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:446)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:109)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:250)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:203)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:435)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.openProject(ProjectManagerImpl.java:410)
	at com.intellij.platform.PlatformProjectOpenProcessor.doOpenProject(PlatformProjectOpenProcessor.java:245)
	at com.intellij.ide.RecentDirectoryProjectsManager.doOpenProject(RecentDirectoryProjectsManager.java:60)
	at com.intellij.ide.RecentProjectsManagerBase.doReopenLastProject(RecentProjectsManagerBase.java:676)
	at com.intellij.ide.RecentProjectsManagerBase$MyAppLifecycleListener.appStarting(RecentProjectsManagerBase.java:718)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:117)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:426)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:387)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:376)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:357)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:43)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:208)
	at com.sun.proxy.$Proxy55.appStarting(Unknown Source)
	at com.intellij.idea.IdeaApplication$IdeStarter.a(IdeaApplication.java:355)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:120)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:122)
	at com.intellij.idea.IdeaApplication$IdeStarter.main(IdeaApplication.java:353)
	at com.intellij.idea.IdeaApplication.a(IdeaApplication.java:198)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:198)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:68)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:413)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: io.reactivex.subjects.Subject PluginClassLoader[reasonml, 0.45-BETA] com.intellij.ide.plugins.cl.PluginClassLoader@a56222b
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:63)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 75 more
```

```
com.reason.ide.RmlProjectTracker@4c9ec70a: io/reactivex/subjects/Subject
java.lang.NoClassDefFoundError: io/reactivex/subjects/Subject
	at com.reason.ide.RmlProjectTracker.projectOpened(RmlProjectTracker.java:23)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.e(ProjectManagerImpl.java:770)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.g(ProjectManagerImpl.java:386)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:165)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:447)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:431)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:415)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.m(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at com.intellij.ide.IdeEventQueue.pumpEventsForHierarchy(IdeEventQueue.java:920)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:215)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runProcessWithProgressSynchronously(ApplicationImpl.java:579)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:446)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:109)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:250)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:203)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:435)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.openProject(ProjectManagerImpl.java:410)
	at com.intellij.platform.PlatformProjectOpenProcessor.doOpenProject(PlatformProjectOpenProcessor.java:245)
	at com.intellij.ide.RecentDirectoryProjectsManager.doOpenProject(RecentDirectoryProjectsManager.java:60)
	at com.intellij.ide.ReopenProjectAction.actionPerformed(ReopenProjectAction.java:72)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.a(ActionMenuItem.java:301)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:307)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:104)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:291)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.a(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:120)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:122)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.JToggleButton$ToggleButtonModel.setPressed(JToggleButton.java:308)
	at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
	at com.apple.laf.ScreenMenuItemCheckbox.itemStateChanged(ScreenMenuItemCheckbox.java:193)
	at java.awt.CheckboxMenuItem.processItemEvent(CheckboxMenuItem.java:389)
	at java.awt.CheckboxMenuItem.processEvent(CheckboxMenuItem.java:357)
	at java.awt.MenuComponent.dispatchEventImpl(MenuComponent.java:355)
	at java.awt.MenuComponent.dispatchEvent(MenuComponent.java:343)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:767)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.m(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: io.reactivex.subjects.Subject PluginClassLoader[reasonml, 0.45-BETA] com.intellij.ide.plugins.cl.PluginClassLoader@a56222b
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:63)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 76 more
```

```
com.reason.ide.RmlProjectTracker@435979c: Argument for @NotNull parameter 'listener' of com/intellij/openapi/editor/impl/event/EditorEventMulticasterImpl.removeDocumentListener must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'listener' of com/intellij/openapi/editor/impl/event/EditorEventMulticasterImpl.removeDocumentListener must not be null
	at com.intellij.openapi.editor.impl.event.EditorEventMulticasterImpl.a(EditorEventMulticasterImpl.java)
	at com.intellij.openapi.editor.impl.event.EditorEventMulticasterImpl.removeDocumentListener(EditorEventMulticasterImpl.java)
	at com.reason.ide.RmlProjectTracker.projectClosed(RmlProjectTracker.java:38)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.o(ProjectManagerImpl.java:791)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:678)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1010)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:675)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.closeAndDispose(ProjectManagerImpl.java:699)
	at com.intellij.ide.impl.ProjectUtil.closeAndDispose(ProjectUtil.java:91)
	at com.intellij.openapi.wm.impl.IdeFrameImpl$3.windowClosing(IdeFrameImpl.java:266)
	at java.awt.AWTEventMulticaster.windowClosing(AWTEventMulticaster.java:349)
	at java.awt.AWTEventMulticaster.windowClosing(AWTEventMulticaster.java:349)
	at java.awt.Window.processWindowEvent(Window.java:2054)
	at javax.swing.JFrame.processWindowEvent(JFrame.java:305)
	at java.awt.Window.processEvent(Window.java:2013)
	at java.awt.Component.dispatchEventImpl(Component.java:4897)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.m(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Thanks, hopefully that helps 👍

**Comment by giraud** [comment]
wasn't expected the classNotFound exceptions :)
thanks for the reports.

But do you have some warn when you update your demo.re file and save it ?
If the plugin can't find the cmi file it should log it

**Comment by giraud** [comment]
Thanks for the reports !!
I wasn't expected the ClassNotFound exception on rx. That explains why you get nothing :(

**Comment by srsholmes** [comment]
Nice, glad that helped. Thanks for getting the reporting in over to me so quickly. 👍

**Comment by srsholmes** [comment]
Hey, thanks for getting back to me. I have actually just taken my Macbook into Apple to get a new screen and wont have it for a few days. I am also off on holiday for a bit so wont have my laptop with me. I will give it a test though first thing when I am back, and let you know how it goes. 👍

**Comment by giraud** [comment]
cool, no hurry. I'll be in holidays also !

**Comment by giraud** [comment]
I've released 0.45.1.
do not use 0.45.

**Comment by srsholmes** [comment]
👍 Hey @giraud hope you had a good holiday. With version 0.47, I am now getting the type definitions in line! 😊 Thanks very much for sorting this out, great work, I will close the issue.
